lang/ilias_es.lang

```diff
@@ -1947,7 +1947,6 @@ auth#:#login_page_switch_rte#:#Usar el editor de texto enriquecido
 auth#:#lti_consumer_inactive#:#El consumidor de herramienta LTI está deshabilitado.
 auth#:#saml_tab_head_idp#:#IDP
 awrn#:#awareness_now_online#:#En-línea ahora
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.